web/elm/src/Dashboard/Grid/Constants.elm

```diff
@@ -21,17 +21,27 @@ cardBodyHeight =
 
 padding : number
 padding =
-    25
+    32
+
+
+carHeaderVerticalSpace : number
+carHeaderVerticalSpace =
+    8
 
 
 cardHeaderHeight : number -> number
 cardHeaderHeight numRows =
-    20 + 30 * numRows
+    32 + 21 * numRows + (numRows - 1) * carHeaderVerticalSpace
+
+
+groupHeaderLineHeight : number
+groupHeaderLineHeight =
+    25
 
 
 groupHeaderHeight : number
 groupHeaderHeight =
-    60
+    padding + groupHeaderLineHeight
 
 
 sectionHeaderHeight : number
```

Comment:
This should be increased - it's now 94px. Perhaps we can split it up into two constants - `sectionHeaderLineHeight: 30px` + `2 * padding`. If we do this, we should use padding in `Dashboard/Styles.elm` for `pipelineSectionHeader`